### PR TITLE
Update harbour-seaprint-pl.ts

### PR DESCRIPTION
**Pull request by karolkurek**
I have created improved polish translation - mostly just filled the blanks.

### CODE REVIEW

File: translations/harbour-seaprint-pl.ts [hunk 1]
```diff
@@ -5406,99 +5406,99 @@
...
+        <translation>góra</translation>
     </message>
     <message>
         <source>stacker</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <source>mailbox</source>
-        <translation type="unfinished"></translation>
+        <translation>skrzynka e-mail</translation>
```
Comment (attah) [inline translations/harbour-seaprint-pl.ts]: I think mail here is dead-tree snail-mail actually :)

**Review by attah — COMMENTED** [review]
One small comment.
Formatting etc looks great, and what little i understand looks sensible.

**Comment by karolkurek** [comment]
I wasn't aware of such a accessory like mailbox. Now its fixed along with the stapler and stacker - I wasn't sure what were they and left them untouched in previous commit.

**Comment by attah** [comment]
Thanks!
I can only assume it goes out for manual sorting somehow, if it is used at all in reality.
I see nothing about stapler in the new commit. 
But i am happy to merge as-is, so just for your information if you wanted to add it.

**Comment by karolkurek** [comment]
> Thanks! I can only assume it goes out for manual sorting somehow, if it is used at all in reality. I see nothing about stapler in the new commit. But i am happy to merge as-is, so just for your information if you wanted to add it.

I am glad to hear that! You are right about the stapler, I was thinking about it but it is nonexistent in 'utils' section.

**Comment by attah** [comment]
Thanks for your contribution!
Are you saying it is missing then, from values of what attribute?

**Comment by karolkurek** [comment]
Everything is fine, nothing is missing. When I was trying to figure out what the 'mailbox' is I found 3 in 1 HP device (mailbox, stapler, stacker) and accidentaly I wrote all those three in a commit comment and in my next message, although stapler was not present in a translation file.

**Pull request merged**